Dockerfile-ubuntu.template

```diff
@@ -25,42 +25,14 @@ ARG PGP_KEYSERVER=keyserver.ubuntu.com
 
 ENV OPENSSL_VERSION {{ .openssl.version }}
 ENV OPENSSL_SOURCE_SHA256="{{ .openssl.sha256 }}"
-# https://www.openssl.org/community/otc.html
 # https://www.openssl.org/source/
 ENV OPENSSL_PGP_KEY_IDS="{{
 [
-	# Dmitry Belyavsky
-
-	# Matt Caswell
-	"8657 ABB2 60F0 56B1 E519 0839 D9C4 D26D 0E60 4491",
-
-	# Paul Dale
-	"B7C1 C143 60F3 53A3 6862 E4D5 231C 84CD DCC6 9C45",
-
-	# Tim Hudson
-	"C1F3 3DD8 CE1D 4CC6 13AF 14DA 9195 C482 41FB F7DD",
-
-	# Hugo Landau
-	"95A9 908D DFA1 6830 BE9F B900 3D30 A3A9 FF13 60DC",
-
-	# Richard Levitte
-	"7953 AC1F BC3D C8B3 B292 393E D5E9 E43F 7DF9 EE8C",
-
-	# Shane Lontis
-
-	# Tomas Mraz
-	"A21F AB74 B008 8AA3 6115 2586 B8EF 1A6B A9DA 2D5C",
-
-	# Kurt Roeckx
-	"E5E5 2560 DD91 C556 DDBD A5D0 2064 C536 41C2 5E5D",
-
-	# Matthias St. Pierre
-
-	# Nicola Tuveri
-
-	# OpenSSL OMC key
-	# https://github.com/openssl/web/pull/415
-	"EFC0 A467 D613 CB83 C7ED 6D30 D894 E2CE 8B3D 79F5",
+	# "OpenSSL <openssl@openssl.org>"
+	# https://openssl-library.org/source/index.html
+	# "The current releases are signed by the OpenSSL key with fingerprint:"
+	# https://keys.openpgp.org/search?q=openssl%40openssl.org
```

Comment:
Since we only appear to use `PGP_KEYSERVER` for OpenSSL, perhaps we should just switch to the superior keys.openpgp.org service explicitly? :eyes: